src/components/Activities.tsx

```diff
@@ -1,9 +1,12 @@
 import Activity from './Activity';
 import Poster from './Poster';
-import PropTypes from 'prop-types';
 import { useActivitiesQuery } from '../store/api';
 
-export default function Activities({ current }) {
+type ActivitiesProps = {
```

Comment:
The state machine props behave like pages or slides. 

I would have created a StateMachineSlideProps kind of interface, which can be reused for quotes, ads and activities. All these 'pages' have the same props